app/controllers/admin/api/services/backend_usages_controller.rb

```diff
@@ -16,7 +16,7 @@ class Admin::Api::Services::BackendApiConfigsController < Admin::Api::Services::
   # swagger
   ##~ sapi = source2swagger.namespace("Account Management API")
   ##~ e = sapi.apis.add
-  ##~ e.path = "/admin/api/services/{service_id}/backend_apis.json"
+  ##~ e.path = "/admin/api/services/{service_id}/backend_usages.json"
```

Comment:
Those routes do not look correct. I think using acceptance spec will ensure we are really have the same path as described in the documentation. Thus my suggestion to just rename the controller

![image](https://user-images.githubusercontent.com/64276/67212760-7e36d200-f425-11e9-901c-75f36fd7058b.png)

Reply:
u do not have the latest code locally, do `git pull`
![image](https://user-images.githubusercontent.com/11318903/67213234-f3060e00-f41d-11e9-9a64-bb72ffe29b44.png)